### PR TITLE
FIX: BBCode parsing specs

### PR DESCRIPTION
**Pull request by ZogStriP**
Requires https://github.com/discourse/discourse/pull/27173 to be merged.

Will add a `.discourse-compatibility` once merged.

### CODE REVIEW

File: spec/integration/post_spec.rb [hunk 1]
```diff
@@ -202,36 +238,20 @@
...
-
-            post = create_post_with_event(user, "url=").reload
-            expect(post.event.url).to be_blank
           end
 
           it "works with status attribute" do
             post = create_post_with_event(user, 'status="private"').reload
-            expect(post.event.status).to eq(DiscoursePostEvent::Event.statuses[:private])
-
-            post = create_post_with_event(user, 'status=""').reload
```
Comment (ZogStriP) [inline spec/integration/post_spec.rb:230]: Removed these specs which aren't supported anymore.

File: spec/integration/post_spec.rb [hunk 2]
```diff
@@ -202,36 +238,20 @@
...
-            expect(post.event.name).to be_blank
           end
 
           it "works with url attribute" do
             url = "https://www.discourse.org"
 
             post = create_post_with_event(user, "url=\"#{url}\"").reload
             expect(post.event.url).to eq(url)
-
-            post = create_post_with_event(user, 'url=""').reload
```
Comment (ZogStriP) [inline spec/integration/post_spec.rb:219]: Removed these specs which aren't supported anymore.

File: spec/integration/post_spec.rb [hunk 3]
```diff
@@ -241,23 +261,38 @@
...
 
             post =
-              create_post_with_event(user, 'status="private" allowedGroups="euro,america"').reload
+              create_post_with_event(
+                user,
+                'status="private" allowedGroups="euro,america"'
+              ).reload
             expect(post.event.raw_invitees).to match_array(%w[euro america])
 
-            post = create_post_with_event(user, 'status="private" allowedGroups=""').reload
```
Comment (ZogStriP) [inline spec/integration/post_spec.rb:257]: Removed these specs which aren't supported anymore.

File: spec/integration/post_spec.rb [hunk 4]
```diff
@@ -241,23 +261,38 @@
...
+            post = create_post_with_event(user, 'status="private"').reload
             expect(post.event.raw_invitees).to eq([])
 
-            post = create_post_with_event(user, 'status="private" allowedGroups=').reload
```
Comment (ZogStriP) [inline spec/integration/post_spec.rb:260]: Removed these specs which aren't supported anymore.

**Review by arpitjalan — APPROVED** [review]

**Pull request merged**